modules/quality/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(the_description "Image Quality Analysis API")
 ocv_define_module(quality opencv_core opencv_imgproc opencv_ml WRAP python)
-ocv_add_testdata(samples/ contrib/quality
-    FILES_MATCHING PATTERN "*.yml"
-)
\ No newline at end of file
+
+# add brisque model, range files to installation.  the tests and quality module use these files unless otherwise directed
+file(GLOB QUALITY_MODEL_DATA samples/*.yml)
+install(FILES ${QUALITY_MODEL_DATA} DESTINATION ${OPENCV_OTHER_INSTALL_PATH}/quality COMPONENT libs)
```

Comment:
Please keep using of `ocv_add_testdata()` macro.
Tests should run from the build directory too (without any installation path).